bugsnag/bugsnag-react-native
# Ensure RCTFatalExceptions are skipped

In the case that JS initialization never happens, move up the essential elements of the JS config to ensure `RCTFatalException` messages are missed. 


Fixes #115 

Steps to test:

* Initialize a React Native app using the docs instructions, skipping the JS portion
* Configure bugsnag
* Run the app on iOS
* Ensure that causing a JS error does not generate an error report with the title `RCTFatalException ...`
